libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.spec.ts

```diff
@@ -1094,27 +1094,57 @@ describe('Autocomplete component', () => {
       await fixture.whenStable();
       await expectAsync(document.body).toBeAccessible(axeConfig);
       fixture.detectChanges();
+      inputElement.focus();
+      SkyAppTestUtility.fireDomEvent(inputElement, 'focus');
       inputElement.value = 'r';
       SkyAppTestUtility.fireDomEvent(inputElement, 'input');
       fixture.detectChanges();
       await fixture.whenStable();
+      expect(getSearchResultsContainer()).toBeTruthy();
+      await expectAsync(document.body).toBeAccessible(axeConfig);
+    });
+
+    it('should be accessible with enableShowMore', async () => {
+      component.enableShowMore = true;
+
+      const axeConfig = {
+        rules: {
+          region: {
+            enabled: false,
+          },
+        },
+      };
+
+      fixture.detectChanges();
+
+      const inputElement: HTMLInputElement = getInputElement();
+
+      await fixture.whenStable();
+      fixture.detectChanges();
+      inputElement.focus();
```

Comment:
This test is intended to test the accessibility of the state where the input has focus and nothing has been entered yet, but show more is enabled. That was a case where the test would break in axe 4.7.

Reply:
The test should still show the dropdown with the "Show more" link, correct? It's not visible when `destroyAfterEach` is set to `false`. 🤔

Reply:
When I set a breakpoint in Chrome, the overlay is there prior to running the `toBeAccessible()` test, but gone when it finishes. Something about the test causes the overlay to dismiss. The test fails without the changes to the component, so I think it's a valid test that verifies the fix.